src/ngsa-typescript/README.md

```diff
@@ -42,89 +45,41 @@ npm start
 # optionally, set the logging level verboseness with --log-level (or -l)
 # 'info' is the default
 
-npm start -- --log-level
+npm start -- --log-level warn
 
 # alternatively you can set the following environment variables and run without command line args
 
 export LOG_LEVEL=info # (optional)
 
 npm start
 
-```
+# wait until 'Server is listening on port 4120' is displayed
 
-wait for `Server is listening on port 4120`
+```
 
-### Testing the application
+## Verify the Web API is running
 
 Open a new bash shell
 
 > Visual Studio Codespaces allows you to open multiple shells by clicking on the `Split Terminal` icon
 
 ```bash
 
-# test the application
+curl http://localhost:4120/version
 
-# test using httpie (installed automatically in Codespaces)
-http localhost:4120/version
-
-# test using curl
-curl localhost:4120/version
-
-```
-
-Stop the app by typing Ctrl-C or the stop button if run via F5
-
-### Deep Testing
-
-We use [Web Validate](https://github.com/Microsoft/webvalidate) to run deep verification tests on the Web API
-
-If you have dotnet core sdk installed
-
-```bash
+curl http://localhost:4120/api/genres
 
-# install Web Validate as a dotnet global tool
-# this is automatically installed in CodeSpaces
-dotnet tool install -g webvalidate
+# from src/ngsa-csharp/Ngsa.LodeRunner
+dotnet run -- -s http://localhost:4120 -f baseline.json
 
-# make sure you are in the root of the repository
-
-# run the validation tests
-# validation tests are located in the TestFiles directory
-cd TestFiles
-
-webv -s localhost:4120 -f baseline.json
-
-# there may be a validation error on the /healthz/ietf endpoint test
-#   json: status: warn : Expected: pass
-# the "warn" status indicates a slower than normal response time
-# this is OK and will occasionally occur
-
-# benchmark.json is a 300 request test that covers the entire API
-
-# cd to root of repo
-cd ..
+# longer end-to-end test
+dotnet run -- -s http://localhost:4120 -f benchmark.json
```

Comment:
additionally, we could use docker run

Seems more natural to use docker as some TS devs might not have dotnet installed?